game/ui/ui-game.js

```diff
@@ -0,0 +1,124 @@
+import { createElement } from './ui-utils.js';
+import UIElement, { UIAnchor } from './ui-element.js';
+import GameManager from '../managers/game-manager.js';
```

Comment:
Should change to attach click listener instead. Game Manager only used once, can further separate elements